### PR TITLE
Website

### PR DESCRIPTION
**Pull request by Sl1mb0**
- Adds an image of Ruperf the Crab
 - Outlines 3 things our tool aims to achieve
 - Provides brief descriptions of other tools and how our tool hopes to add the existing landscape
 - Adds brief outlines of other sections that I will fill out later today.

### CODE REVIEW

**Review by Boursler — APPROVED** [review]
Hilarious read, thank you for doing this.

**Comment by kagutaba256** [comment]
He lives... 👀👀 Excellent work, these sections all seem good to include.

**Review by kagutaba256 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+<br>
+# Introduction 
+
+All software should perform at a high level. That means going underneath the hood, 
+locating areas of improvement, and fine tuning the machine. 
+Hardware instructions, computer cycles, cache-accesses,
+and context switches; how much memory is being used or how long a process takes; 
+these and more, are all things to observe when seeking insight into where and how to improve performance.
+
+`ruperf` was borne out of a desire to achieve three things:
```
Comment (ScherrerMichael) [inline index.html]: Was wondering if `"borne"` was intentional or not.

**Review by ScherrerMichael — CHANGES_REQUESTED** [review]
Made a note about a single words spelling inline. First time doing an inline suggestion, I hope it shows up in this review. :)

Edit: Ah it separates the comments, cool! Looks great!

**Review by ScherrerMichael — APPROVED** [review]

**Pull request merged**